extensions/amp-analytics/0.1/vendors.js

```diff
@@ -1286,12 +1287,24 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
         'date=${timestamp}&' +
         'ampid=${clientId(_parsely_visitor)}',
       'pageview': '${basePrefix}&action=pageview',
+      'heartbeat': '${basePrefix}&action=heartbeat' +
+      '&tt=${totalEngagedTime}&inc=${incrementalEngagedTime(parsely-js)}',
     },
     'triggers': {
       'defaultPageview': {
         'on': 'visible',
         'request': 'pageview',
       },
+      'heartbeat': {
+        'on': 'timer',
+        'timerSpec': {
+          'interval': 5,
+          'vars': {
+            'name': 'parsely-js',
```

Comment:
Again I see you are using  `${incrementalEngagedTime(parsely-js)}`, in that case there's no need to specify the `name` var here.